server/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -56,3 +56,8 @@ semantics.previousWithoutPicFor88=A conditional data item is not found for this
 semantics.tooManyClauses=A duplicate %s clause was found in a data description entry
 semantics.globalNon01Level=GLOBAL can only be used on level 01
 semantics.globalTooManyDefinitions=Global variable must have a unique name
+statements.invalidReceivingField=Invalid receiving field type. Expected: %s
+statements.invalidSendingField=Invalid sending field type. Expected: %s
+variables.indexName=Index name
+variables.elementaryInteger=Elementary integer data item
+variables.nonzeroInteger=Nonzero integer
```

Comment:
Optional: Non-zero

Reply:
I agree with non-zero

Reply:
I also agree, but the specification calls it "nonzero" http://publibfp.boulder.ibm.com/epubs/pdf/igy6lr30.pdf on page 433 in the "Format 2: SET for adjusting indexes" paragraph